### PR TITLE
binkd 1.0.4 (new formula)

### PR DESCRIPTION
**Pull request by dmirkitanov**
Binkd is a Fidonet mailer which works over tcp/ip, homepage: http://binkd.grumbler.org/

### CODE REVIEW

File: Library/Formula/binkd.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class Binkd < Formula
+  desc "TCP/IP FTN Mailer"
+  homepage "http://binkd.grumbler.org/"
+  url "ftp://happy.kiev.ua/pub/fidosoft/mailer/binkd/binkd-1.0.4.tar.gz"
+  sha256 "917e45c379bbd1a140d1fe43179a591f1b2ec4004b236d6e0c4680be8f1a0dc0"
+
+  def install
+    cp Pathname.glob("mkfls/unix/*").select(&:file?), "."
+    inreplace "binkd.conf", "/var/", "/usr/local/var/"
```
Comment (bfontaine) [inline Library/Formula/binkd.rb]: Use `inreplace "binkd.conf", "/var/", "#{var}/"` instead.

File: Library/Formula/binkd.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+class Binkd < Formula
+  desc "TCP/IP FTN Mailer"
+  homepage "http://binkd.grumbler.org/"
+  url "ftp://happy.kiev.ua/pub/fidosoft/mailer/binkd/binkd-1.0.4.tar.gz"
+  sha256 "917e45c379bbd1a140d1fe43179a591f1b2ec4004b236d6e0c4680be8f1a0dc0"
+
+  def install
+    cp Pathname.glob("mkfls/unix/*").select(&:file?), "."
```
Comment (bfontaine) [inline Library/Formula/binkd.rb]: Use `Dir["mkfls/unix/*"]` instead of `Pathname.glob`. Why is this step necessary?

Comment (dmirkitanov) [inline Library/Formula/binkd.rb]: Authors keep makefiles separately for each platform, and to configure/make these files should be moved to the root folder of the project, so this is the way they've made the building process.

**Comment by dmirkitanov** [comment]
Fixed the issues.

**Comment by mikemcquaid** [comment]
Thanks @dmirkitanov!

**Pull request closed without merging**